term_bsd.go

```diff
@@ -1,3 +1,4 @@
+// +build darwin freebsd
```

Comment:
http://golang.org/pkg/go/build/ specifies that "To distinguish build constraints from package documentation, a series of build constraints must be followed by a blank line."

So if no newline is there then it can't tell if it's a comment or a build constraint, and the comment is ignored.